documentation/service-rules/appropriate-body-users.md

```diff
@@ -116,7 +116,7 @@ The registration process takes place with the following steps:
   - must be present
   - must be after the QTS award date
 * **Induction programme**
-  - must be either `fip`, `cip` or `diy` (enforced in the database)
+  - must be either `provider_led` or `school_led` (enforced in the database)
```

Comment:
can we keep this as is, but add a comment that aesthetically we show provider/school led instead